docs/UserGuide.md

```diff
@@ -60,13 +30,13 @@ Taskmania (based off AB3) is a **desktop app for a project leader to manage team
 4. Double-click the file to start the app. The window that appears will be similar to the below should appear in a few seconds. Note how
  the app contains some sample information.<br>
    ![Ui](images/Ui.png)
-   *Figure 1: What the app looks like on startup*
+   *Figure 1: A preview of Taskmania in action*
```

Comment:
in action implies user is halfway using the app, the figure shows what the user shld see when the user first opens the app